_maps/kilostation.json

```diff
@@ -9,5 +9,10 @@
 	    "ferry": "ferry_kilo",
 	    "cargo": "cargo_kilo",
 	    "whiteship": "whiteship_kilo"
-    }
+    },
+	"job_changes": {
+		"cook": {
+			"additional_cqc_areas": ["/area/service/bar/atrium"]
```

Comment:
Are we sure this is the way to go?

I was expecting just a map helper entity you plop one of in each area that the chef should have CQC in, that's much more intuitive for mappers than editing JSON files, I feel.

Reply:
Why is the normal bar not on this list?

Reply:
> 
> 
> Are we sure this is the way to go?
> 
> I was expecting just a map helper entity you plop one of in each area that the chef should have CQC in, that's much more intuitive for mappers than editing JSON files, I feel.

It's a lot more clear which areas the chef has CQC in when it's pulled out into its own config.

I felt like adding more mapping objects served to obfuscate exactly where a chef could perform CQC on any given. I personally think it makes sense to keep it in a mapping config where it can be easily be seen and changes tracked better.

Reply:
For context, we're starting to get 
![image](https://user-images.githubusercontent.com/24975989/110602205-a7bcde80-817d-11eb-9ffe-09a8731a88f7.png)
an awful lot of clutter
![image](https://user-images.githubusercontent.com/24975989/110602275-bc00db80-817d-11eb-9626-00ea3cded99a.png)
when it comes to mapping helpers
![image](https://user-images.githubusercontent.com/24975989/110602323-cc18bb00-817d-11eb-8c01-ee9a7f6133f6.png)